packages/calypso-products/src/plans-list.tsx

```diff
@@ -1168,7 +1168,7 @@ const getPlanJetpackSecurityT1Details = (): IncompleteJetpackPlan => ( {
 		),
 	getLightboxDescription: () =>
 		translate(
-			'Easy-to-use, comprehensive WordPress site security including backups, malware scanning, and spam protection.'
+			'Easy-to-use, comprehensive WordPress site security including backups, malware scanning, and spam protection. Includes VaultPress Backup, Jetpack Scan, and Akismet Anti-spam.'
```

Comment:
I feel like the message here is redundant `including backups, malware scanning, and spam protection. Includes VaultPress Backup, Jetpack Scan, and Akismet Anti-spam.`

Should we just simplify it like below?

```
Easy-to-use, comprehensive WordPress site security including
VaultPress Backup, Jetpack Scan, and Akismet Anti-spam.
```

Reply:
Good call. I've added a line break, I think it helps.